README.md

```diff
@@ -108,7 +108,7 @@ await fastify.ready()
  | transformStaticCSP   | undefined        | Synchronous function to transform CSP header for static resources if the header has been previously set.                  |
  | transformSpecification     | undefined        | Synchronous function to transform the swagger document.                                                                   |
  | transformSpecificationClone| true             | Provide a deepcloned swaggerObject to transformSpecification                                                                    |
- | uiConfig             | {}               | Configuration options for [Swagger UI](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md). Must be literal values, see [#5710](https://github.com/swagger-api/swagger-ui/issues/5710).                                                                                                            |
+ | uiConfig             | {}               | Configuration options for [Swagger UI](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md). Functions should be self contained, as variables outside the functions will not work. You can access the swagger-ui element with `ui` as it is assigned to `window.ui`.                                                                                                   |
```

Comment:
Could we add a reference to a test example?

I'm afraid this will become an endless request of examples

Reply:
uiConfig with onComplete where we do an alert?

Something like this?

```js
'use strict'

const fastify = require('fastify')({ logger: true })

fastify.register(require('@fastify/swagger'), {
  mode: 'static',
  specification: {
    path: './examples/example-static-specification.json'
  }
})

fastify.register(require('../index'), {
  uiConfig: {
    onComplete: () => { alert('test') }
  }
})

fastify.listen({ port: 3000 }, (err) => {
  if (err) throw err
})

```

Reply:
Yes - and/or 

```
initOAuth: {
      scopes: ['openid', 'profile', 'email', 'offline_access']
    }
```

if the test inject a request to the new endpoint we can just check the the data are rawly serialized and the flow is complete and working

Reply:
Yeah. I will probably also add some section into the readme.